File: src/main/java/com/application/lamion/repository/RequestAnalyticsRepository.java
package com.application.lamion.repository;

import com.application.lamion.model.RequestAnalytics;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RequestAnalyticsRepository extends JpaRepository<RequestAnalytics, Long> {
    List<RequestAnalytics> findAllByEventIdOrderByCountDesc(long eventId);
}

File: server/src/main/java/com/application/lamion/repository/RequestAnalyticsRepository.kt
package com.application.lamion.repository

import com.application.lamion.model.RequestAnalytics
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RequestAnalyticsRepository : JpaRepository<RequestAnalytics?, Long?> {
    fun findAllByEventIdOrderByCountDesc(eventId: Long): List<RequestAnalytics?>?
}
